chaoscenter/web/.husky/pre-commit

```diff
@@ -1,8 +1,9 @@
 #!/bin/sh
 "$(dirname "$0")/_/husky.sh"
 
-MATCH_PATH='chaoscenter/web/'
+echo "\033[0;31;1mYou can't commit directly to main branch\033[0m"
```

Comment:
is this line correct? this will be printed for each pre-commit hook irrespective to the branch